### ISSUE TITLE
Color contrast of links in Feature Split

### ISSUE THREAD

**Issue by magjarvis** [ATLAS-Illinois/Illinois-Framework-Theme#1142]
There are instances of text that’s below minimum color contrast ratios. The white text against a bright orange background does not meet minimum color contrast requirements, the hover/visited behavior of links on the blue gradient background also does not meet contrast requirements

1. Create a Feature Split paragraph
2. Choose either an Orange or Blue Gradient for the background color or theme
3. Add a link in the body field

**Expected behavior**
See this link https://brand.illinois.edu/visual-identity/color#web-color-pairings-for-accessibility for accessible color combinations and ensure that the Feature Split links match these themes

<img width="1202" height="627" alt="Image" src="https://github.com/user-attachments/assets/67bb5afe-8fa3-4eb9-9336-2f7b44e31b75" />

**Comment by trubach** [comment]
Natalie will investigate if this is an issue in the ILFW 5.x branch

**Comment by nhjohnstone** [comment]
The semantic colors are working for the links in 5.x branch, this is all good.

<img width="896" height="558" alt="Image" src="https://github.com/user-attachments/assets/695ab682-9301-4e90-9d8f-00e09a5173a0" />

<img width="1039" height="531" alt="Image" src="https://github.com/user-attachments/assets/5c149549-174d-4849-a0f6-e2174c7d9d34" />

**Comment by tjackson2113** [comment]
Added 4.x version, removed 5.x as it's fine in 5.

**Comment by nhjohnstone** [comment]
@magjarvis, for 4.x, I fixed the link color for the blue gradient. I am not seeing any issue with the links on the orange gradient. Nothing is getting flagged for me in wave. If its the white text on the orange button, that is also passing the web aim color checker. Let me know if you have more info on the links on the orange gradient, I might be missing something.

<img width="799" height="934" alt="Image" src="https://github.com/user-attachments/assets/afbe5bb7-89c0-4823-86b2-1feb524afe9a" />
